### DISCUSSION TITLE
layui.js:1 Refused to apply inline style

### DISCUSSION THREAD

**Discussion by bageheyalu** [layui/layui#1730] [问答]
CSP不能设置unsafe inline，程序定位在table.render 语句处报错
”“”
Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'nonce-EDNnf03nceIOfn39fn3e9h3sdfa' https://fonts.googleapis.com". Either the 'unsafe-inline' keyword, a hash ('sha256-6oq7QMcgErSaWQMXh69cvsRxJ7/m9FiEzuBolnO5m30='), or a nonce ('nonce-...') is required to enable inline execution.
“”“
定位到代码处为
”“”
    table.render({
        limit: 10,
        id:'acl-table',
        skin: 'line',
        height: 'full-150',
        elem: '#acl-table',
        toolbar: '#acl-table-toolbar',
        page: true, //开启分页
        cols: [.....
“”“
已经尝试将layui从2.6.8升级到2.9.7，依旧报错

**Comment by sentsim** [comment]
参考：https://gitee.com/layui/layui/issues/I7CKHN#note_18758029_link
